### PR TITLE
first upgrades

### PR DESCRIPTION
**Pull request by skymandr**
This is a part of #6

Here I upgrade dev-dependencies as far as I can without breaking anything. There is a lot left to do here though, as well as with the non-dev dependencies.

The most important thing in this upgrade is probably, that we can bump the node image used from 8 to 16.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,37 +2,38 @@
...
-    "babel-plugin-istanbul": "^4.1.5",
+    "babel-plugin-istanbul": "^4.1.6",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
-    "css-loader": "^0.28.7",
+    "css-loader": "^5.0.0",
     "enzyme": "^3.1.0",
     "enzyme-adapter-react-16": "^1.0.2",
-    "eslint": "^4.11.0",
+    "eslint": "^4.19.1",
```
Comment (skymandr) [inline package.json:18]: This is crucial. It could be further updated, but `5.x.x` requires an update of `eslint-config-airbnb` which in turn causes a lot of lint issues, that would make this PR very messy. I intend to try this, but would prefer to do it in a separate PR.

File: package.json [hunk 2]
```diff
@@ -2,37 +2,38 @@
   "scripts": {
     "test": "BABEL_ENV=test karma start --single-run",
     "build": "webpack",
-    "lint": "eslint --ext mjs,jsx,js scanomatic/ui_server_data/js/ccc scanomatic/ui_server_data/js/specs"
+    "lint": "eslint --ext mjs,jsx,js scanomatic/ui_server_data/js/ccc scanomatic/ui_server_data/js/tests"
```
Comment (skymandr) [inline package.json:5]: Oops!

**Review by local-minimum — COMMENTED** [review]
Not much to say but I guessed you've brought up SoM and tested interacting with the various views to see that everything looks 🆗

**Comment by skymandr** [comment]
> Not much to say but I guessed you've brought up SoM and tested interacting with the various views to see that everything looks ok

Yes, and as far as I can tell it looks ok. It will be more annoying to check when we get to the actual deps...

**Review by local-minimum — APPROVED** [review]
